script/test/cmd/tests.sh

```diff
@@ -76,9 +76,11 @@ convert::expect_success "kompose --provider=openshift -f $KOMPOSE_ROOT/script/te
 ######
 # Tests related to docker-compose file in /script/test/fixtures/volume-mounts/simple-vol-mounts
 # kubernetes test
-convert::expect_success "kompose -f $KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/docker-compose.yml convert --stdout -j" "$KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/output-k8s.json"
+convert::expect_success_and_warning "kompose -f $KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/docker-compose.yml convert --stdout -j" "$KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/output-k8s.json" "volume ""httpd-claim0"" of size ""100 Mi"" created, PersistentVolume should be created before in order to PersistentVolumeClaim start working, unless it will be in pending state
+ If Cluster has dynamic Provisioning and Storage Classes, So you don't have to do anything"
 # openshift test
-convert::expect_success "kompose --provider=openshift -f $KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/docker-compose.yml convert --stdout -j" "$KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/output-os.json"
+convert::expect_success_and_warning "kompose --provider=openshift -f $KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/docker-compose.yml convert --stdout -j" "$KOMPOSE_ROOT/script/test/fixtures/volume-mounts/simple-vol-mounts/output-os.json" "volume ""httpd-claim0"" of size ""100 Mi"" created, PersistentVolume should be created before in order to PersistentVolumeClaim start working, unless it will be in pending state
+ If Cluster has dynamic Provisioning and Storage Classes, So you don't have to do anything"
```

Comment:
No need to write any tests for this, this is just a log output, no change to logic in code.